lib/tasks/populate_newposition.rake

```diff
@@ -0,0 +1,17 @@
+desc 'Populate new position column'
+task populate_newposition: :environment do
```

Comment:
Can we add some logging around this task t make it clearer when it began, finished, and what it affected?

Reply:
Done